t/timezones.t

```diff
@@ -0,0 +1,32 @@
+use strict;
+use warnings;
+use lib 't/inc';
+use nptestutils;
+
+use Number::Phone;
+use Number::Phone::Lib;
+use Number::Phone::NANP;
+use Test::More;
+
+my %tests = (
+    '+442087712924' => ['Europe/London'], # Geographic UK number
+    '+445511000000' => ['Europe/Guernsey','Europe/Isle_of_Man','Europe/Jersey','Europe/London'], # Non-geographic UK number
+    '+12024181440'  => ['America/New_York'], # geographic New York number
+    '+18765551234'  => ['America/Jamaica'], # geographic Jamaican number
```

Comment:
It's probably worth having a test for a NANP island country with only one area code, such as Anguilla (+1 264), and also tests for each of the area codes for an island like Jamaica, which has two. May be easier to use Puerto Rico (+1 787 and +1 939) though, as it's hard to find a live number in Jamaica's second area code at the moment.

Reply:
Added some more unit tests in https://github.com/DrHyde/perl-modules-Number-Phone/pull/134/commits/825dc3fcb09b2537beaab472b6b93428921547fb